test/unit/org/apache/cassandra/cql3/KeyCacheCqlTest.java

```diff
@@ -262,7 +262,7 @@ private void test2iKeyCachePaths() throws Throwable
         long hits = metrics.hits.getCount();
         long requests = metrics.requests.getCount();
         assertEquals(0, hits);
-        assertEquals(210, requests);
+        assertEquals(206, requests);
```

Comment:
@blerer, knowing you I am pretty sure you checked this formula, also the number goes down in the right direction, but we talked with @maedhroz that it would be great to add a comment on it, how did we receive this number

Reply:
The expected number for queries without indices = previous number of request + Sum for each request of (1 (the row is only in one sstable) + number of bloom filter false positive for the request)

For indices queries you have to add the index query + the number of bloom filter false positive for index query

I will try to find a way to make it explicite in the test.